script/apps/Aries/Creator/Game/Mod/ModBase.lua

```diff
@@ -124,4 +128,108 @@ function ModBase:RegisterCommand(name, cmd)
 	local Commands = commonlib.gettable("MyCompany.Aries.Game.Commands");
 	Commands[name] = cmd;
 end
+
+-- set any kind of data
+-- @param dataBundle: bundles of data
+-- @param dataTable: table data type
+-- e.g self:SetWorldData("mydata",{hello="world",thank="you"})
+function ModBase:SetWorldData(dataBundle,dataTable)
+	if(self.worldData == nil) then
+		local modName   = self.Name;
+		local worldName = GameLogic.GetWorldDirectory();
+		local filePath  = worldName .. "mod/" .. modName .. ".xml";
+
+		self.worldData = ParaXML.LuaXML_ParseFile(filePath);
+	end
+
+	if(self.worldData) then
+		--LOG.std(nil,"debug","xmlFileExist");
+		--If exist the same dataBundle,delete it.
+		for key,value in pairs(self.worldData[1]) do
+			if(type(value) == "table" and value.name == dataBundle) then
+				self.worldData[1][key] = nil;
+			end
+		end
+	else
+		--LOG.std(nil,"debug","xmlFileNotExist");
+		self.worldData = {{name=modName}};
+		ParaIO.CreateDirectory(worldName .. "mod/");
+	end
+
+	local count = 0;
+	for key,value in pairs(self.worldData[1]) do
+		if(type(key) == "number") then
+			count = count + 1;
+		end
+	end
+
+	if(type(dataTable) ~= "table")then
+		LOG.std(nil,"info","dataTable must be table");
+		return false;
+	end
+
+	local xmlTable = {};
+
+	for name,value in pairs(dataTable) do
```

Comment:
also need to support nested table.  so 
you only need to serialize table to string and deserialize